README

```diff
@@ -48,6 +48,13 @@ Git repository for work-in-progress changes is available at:
 <https://github.com/01org/libyami>
 
 
+Test Applications
+-----------------
+To avoid add too many dependency to your project, we split all test applications
+to yami-utils. It include yamidecode/yamiencode and conformance test script.
+You can get it from https://github.com/01org/libyami-utils
+
+
```

Comment:
Good suggestion, your version is better. But next section is important too, it give user preview of decode api. Because yamidecode maybe to complex to first time user.